src/components/pages/lessons-page/components/LessonCard/Lesson.test.js

```diff
@@ -1,8 +1,9 @@
-import { render } from '@testing-library/react';
+import { fireEvent, render, screen } from '@testing-library/react';
 
 import '@testing-library/jest-dom';
 
 import LessonCard from './LessonCard';
+import { act } from 'react-dom/test-utils';
 
 describe('VideoCard component ', () => {
```

Comment:
That`s probably a typo, since you only use LessonCard component here, not VideoCard)